### PR TITLE
Logstash for CloudWatch Alarms

### PR DESCRIPTION
**Pull request by imomou**
Hi Bit-Cloudeders

This is a logstash processing to ElasticSearch specifically for SNS CloudWatch Alarms, I think you will find this useful.

Regards
Imomou

### CODE REVIEW

File: analytics/ls-aws-sqs-default/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM docker.elastic.co/logstash/logstash:6.5.1
```
Comment (SleeperSmith) [inline analytics/ls-aws-sqs-default/Dockerfile]: Please change to using logstash-oss
https://github.com/Bit-Clouded/Angostura/blob/8dc75b5b7fcfbc7e0d29454140afaf9b79ef5f5c/analytics/ls-aws-kinesis/Dockerfile#L1

File: analytics/ls-aws-sqs-default/ls-aws-sqs3-default.conf [hunk 2]
```diff
@@ -0,0 +1,40 @@
+input {
+	sqs {
+		queue => "{{sqs-name}}"
+		region => "{{aws-region}}"
+	}
+}
+
+filter {
+	
+		mutate {
```
Comment (SleeperSmith) [inline analytics/ls-aws-sqs-default/ls-aws-sqs3-default.conf]: This shouldn't be indented to second level. As well as the rest of this section

File: analytics/ls-aws-sqs-default/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+USER root
+
+RUN bin/logstash-plugin install --no-verify
```
Comment (SleeperSmith) [inline analytics/ls-aws-sqs-default/Dockerfile]: Please remove this line.

**Review by SleeperSmith — CHANGES_REQUESTED** [review]

**Review by SleeperSmith — APPROVED** [review]

**Comment by imomou** [comment]
Hi, about the CLA, the root user actually comes from me (imomou), encase you're wondering why they coming from

**Pull request merged**